notebooks/arb_dao_grant_distribution/STIP_Updates/12-1-2024/12-1-2024.md

```diff
@@ -0,0 +1,86 @@
+As per the STIP rules all protocols must post bi-weekly forum updates. The second updates will be due by this Friday, January 12th at 12:00PM EST
+
+# Balancer Bi-Weekly Update 12-1-2024
+![arb-forum-update-4.png](arb-forum-update-4.png)
+## Recap of the Previous Two Weeks
+
+
+**ARB Received Last Disbursement:** 342,858 (KYC delays resulted in collecting 2 biweekly payments at the start of the program, details in change section below)
+
+**ARB Utilized as Incentives in the Last Two Weeks: [205,249](https://arbiscan.io/tx/0x47652a14eebd36fbcf03da6028c49a3a47f39becfa737efb09cec53f79b065eb)
+
+**Contracts incentivized over the last 2 weeks:** A CSV of incentived along with details about $ARB flows to each one and how they were calculated can be found [here](https://github.com/BalancerMaxis/STIP_automation/blob/last-round-2023/output/dao_grant_2023-12-14_2023-12-28.csv).
+These amounts of payment were queued and the required 171,429 $ARB sent for payment by this [multisig transaction](https://app.onchainden.com/safes/arb1:0xb6BfF54589f269E248f99D5956f1fDD5b014D50e/transactions/0x3fc881080230ed0608abfa36d82490fbcf9022d57f6247bacb83b48a21907c27) executed at 15:01 UTC on Friday Decemeber 29th.
+These payments were be streamed to stakers in the gauge by the injector and the gauge system for 2 weeks, ending today on January 12th.
+![Arb_STIP_Gauges_12_28.png](Arb_STIP_Gauges_12_28.png)
+
+
+**Contract address label [Form](https://docs.google.com/forms/d/e/1FAIpQLSd2AYnjAaQjVOLtvemZpsWoN5sTJEJ8dLqdRDExTBQv_SUeug/viewform) completed for all addresses:** Yes
+
+**ARB left over:** 68,571.6
```

Comment:
There's 152k $ARB in the kyc safe now after last round.  This round we're claiming 171k and paying out 82k, so this number should either be the current balance and/or the balance after we run this which will be greater than 152k.